frontend/components/homepage/route.tsx

```diff
@@ -65,3 +66,27 @@ export const SocialRoute = {
   FaceBook,
   Instagram,
 };
+
+export const HomePageContent = {
+  background: {
+    name: "homepage-background",
+    path: "https://s2.loli.net/2024/09/05/mjv9A8IeqlTwDyi.png",
+  },
+  dish: {
+    name: "homepage-dish",
+    path: "https://s2.loli.net/2024/09/05/2sTNeCwxcI4alPZ.png",
+  },
+  title: {
+    name: "homepage-title",
+    text: "Crafted with Tradition, Served with Care: Experience the Authentic Flavors of China at Nanyuan",
+    buttonText: "SEE OUR MENU",
```

Comment:
Since the file name is route, it maybe better to only store info related to the route. If you want to store like text or buttonText maybe you can create a customized hook or just stoke the info within the component.